tests/unit_tests/model/test_telescope_model.py

```diff
@@ -15,226 +15,231 @@
-    tel.changeParameter("camera_config_file", tel.getParameterValue("camera_config_file"))
+    tel.change_parameter("camera_config_file", tel.get_parameter_value("camera_config_file"))
     logger.debug(
-        "tel._isExportedModelFiles should be False because a parameter that "
+        "tel._is_exported_model_files should be False because a parameter that "
         "is a file was changed."
     )
-    assert False is tel._isExportedModelFilesUpToDate
+    assert False is tel._is_exported_model_files_up_to_date
 
 
 def test_load_reference_data(telescope_model_lst):
 
-    telModel = telescope_model_lst
+    tel_model = telescope_model_lst
 
-    assert telModel.referenceData["nsb_reference_value"]["Value"] == pytest.approx(0.24)
+    assert tel_model.reference_data["nsb_reference_value"]["Value"] == pytest.approx(0.24)
 
 
 def test_export_derived_files(telescope_model_lst):
 
-    telModel = telescope_model_lst
+    tel_model = telescope_model_lst
 
-    _ = telModel.derived
+    _ = tel_model.derived
     assert (
-        telModel.getDerivedDirectory()
+        tel_model.get_derived_directory()
         .joinpath("ray-tracing-North-LST-1-d10.0-za20.0_validate_optics.ecsv")
         .exists()
     )
 
 
 def test_get_on_axis_eff_optical_area(telescope_model_lst):
 
-    telModel = telescope_model_lst
+    tel_model = telescope_model_lst
 
-    assert telModel.getOnAxisEffOpticalArea().value == pytest.approx(
+    assert tel_model.get_on_axis_eff_optical_area().value == pytest.approx(
         365.48310154491
     )  # Value for LST -1
 
 
 def test_read_two_dim_wavelength_angle(telescope_model_sst):
 
-    telModel = telescope_model_sst
-    telModel.exportConfigFile()
+    tel_model = telescope_model_sst
+    tel_model.export_config_file()
 
-    twoDimFile = telModel.getParameterValue("camera_filter")
-    assert telModel.getConfigDirectory().joinpath(twoDimFile).exists()
-    twoDimDist = telModel.readTwoDimWavelengthAngle(twoDimFile)
-    assert len(twoDimDist["Wavelength"]) > 0
-    assert len(twoDimDist["Angle"]) > 0
-    assert len(twoDimDist["z"]) > 0
-    assert twoDimDist["Wavelength"][4] == pytest.approx(300)
-    assert twoDimDist["Angle"][4] == pytest.approx(28)
-    assert twoDimDist["z"][4][4] == pytest.approx(0.985199988)
+    two_dim_file = tel_model.get_parameter_value("camera_filter")
+    assert tel_model.get_config_directory().joinpath(two_dim_file).exists()
+    two_dim_dist = tel_model.read_two_dim_wavelength_angle(two_dim_file)
+    assert len(two_dim_dist["Wavelength"]) > 0
+    assert len(two_dim_dist["Angle"]) > 0
+    assert len(two_dim_dist["z"]) > 0
+    assert two_dim_dist["Wavelength"][4] == pytest.approx(300)
+    assert two_dim_dist["Angle"][4] == pytest.approx(28)
+    assert two_dim_dist["z"][4][4] == pytest.approx(0.985199988)
 
 
 def test_read_incidence_angle_distribution(telescope_model_sst):
 
-    telModel = telescope_model_sst
+    tel_model = telescope_model_sst
```

Comment:
Sometimes we use sst and sometimes SST (same for the other telescope types). Shall we agree on one or the other ? I would be in favor of uppercase

Reply:
Sure, indeed it should be uppercase. I will try to find all cases where it's lower case and fix them.

Reply:
Actually, rethinking about it, I think I prefer to keep it in lower case. It fits better with the rest of the code and I think that's more important than the fact it's an acronym. Whenever we interact with the user it is capitalised and I think that's good.

Reply:
I agree with your choice and commented also in other places where we should change to keep it consistent